tests/jenkins/ansible-advanced-homework.groovy

```diff
@@ -0,0 +1,278 @@
+// -------------- Configuration --------------
+// CloudForms
+def opentlc_creds = 'b93d2da4-c2b7-45b5-bf3b-ee2c08c6368e'
+def opentlc_admin_creds = '73b84287-8feb-478a-b1f2-345fd0a1af47'
+def cf_uri = 'https://labs.opentlc.com'
+def cf_group = 'opentlc-access-cicd'
+// IMAP
+def imap_creds = 'd8762f05-ca66-4364-adf2-bc3ce1dca16c'
+def imap_server = 'imap.gmail.com'
+// Notifications
+def notification_email = 'gptezabbixalert@redhat.com'
+def rocketchat_hook = '5d28935e-f7ca-4b11-8b8e-d7a7161a013a'
+
+// SSH key
+def ssh_creds = '15e1788b-ed3c-4b18-8115-574045f32ce4'
+
+// Admin host ssh location is in a credential too
+def ssh_admin_host = 'admin-host-na'
+
+// state variables
+def guid=''
+def ssh_location = ''
+
+
+// Catalog items
+def choices = [
+    'OPENTLC Automation / Ansible Advanced - Homework',
+    'OPENTLC Automation / DEV Ansible Advanced - Homework',
+].join("\n")
+
+def region_choice = [
+    'na',
+    'apac',
+    'emea',
+].join("\n")
+
+pipeline {
+    agent any
+
+    options {
+        buildDiscarder(logRotator(daysToKeepStr: '30'))
+    }
+
+    parameters {
+        booleanParam(
+            defaultValue: false,
+            description: 'wait for user input before deleting the environment',
+                name: 'confirm_before_delete'
+        )
+        choice(
+            choices: choices,
+            description: 'Catalog item',
+            name: 'catalog_item',
+        )
+        choice(
+            choices: region_choice,
+            description: 'Catalog item',
+            name: 'region',
+        )
+    }
+
+    stages {
+        stage('order from CF') {
+            environment {
+                uri = "${cf_uri}"
+                credentials = credentials("${opentlc_creds}")
+                DEBUG = 'true'
+            }
+            /* This step use the order_svc_guid.sh script to order
+             a service from CloudForms */
+            steps {
+                git url: 'https://github.com/fridim/cloudforms-oob'
```

Comment:
you can use the new URL
https://github.com/redhat-gpte-devopsautomation/cloudforms-oob

Reply:
Fixed [3fe6967](https://github.com/redhat-cop/agnosticd/pull/939/commits/3fe696786147d16fedb1c5564337d249fa17db58)